test/passing/gen/gen.ml

```diff
@@ -51,20 +59,24 @@ let emit_test test_name setup =
     match setup.base_file with Some n -> n | None -> test_name
   in
   let extra_deps = String.concat " " setup.extra_deps in
+  let cmd_prefix = if setup.should_fail then "! " else "" in
+  let cmd =
+    Printf.sprintf "%%{bin:ocamlformat}%s %%{dep:%s}" opts base_test_name
```

Comment:
`cmd_prefix` could be printed here instead, so there is less arguments to the printf below.
Also, this could be calling `fprintf` and be a `%a` in the format string below to avoid the intermediate string:
```ocaml
let cmd pp () = ...
```

Reply:
This is not performance sensitive, so I did this to move arguments closer to their specification. Anyway, I reverted to how the code was looking originally, that works.